presto-product-tests/src/modernizer/exclusions.txt

```diff
@@ -0,0 +1 @@
+com/google/inject/Inject
```

Comment:
Looks like it might be possible to use `OptionalBinder`: http://google.github.io/guice/api-docs/4.0/javadoc/com/google/inject/multibindings/OptionalBinder.html